File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/BrowseEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class BrowseEndpoint{

	@SerializedName("browseId")
	private String browseId;

	@SerializedName("params")
	private String params;

	public void setBrowseId(String browseId){
		this.browseId = browseId;
	}

	public String getBrowseId(){
		return browseId;
	}

	public void setParams(String params){
		this.params = params;
	}

	public String getParams(){
		return params;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/BrowseEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class BrowseEndpoint : Parcelable, Serializable {
    @SerializedName("browseId")
    var browseId: String? = null

    @SerializedName("params")
    var params: String? = null
    override fun toString(): String {
        return "BrowseEndpoint{" +
                "browseId = '" + browseId + '\'' +
                ",params = '" + params + '\'' +
                "}"
    }
}